openhands/controller/agent_controller.py

```diff
@@ -87,6 +88,7 @@ def __init__(
         is_delegate: bool = False,
         headless_mode: bool = True,
         status_callback: Callable | None = None,
+        context_window: int | None = None,
```

Comment:
Maybe we can use [this setting](https://github.com/All-Hands-AI/OpenHands/blob/2b7932b46c37aec05e867e5a4b7f724e41c6f89a/openhands/core/config/llm_config.py#L35), what do you think?

Reply:
great, will use.

Reply:
threw it in: [55a1f5c](https://github.com/All-Hands-AI/OpenHands/pull/5079/commits/55a1f5cad058a769b9be0e6397a0138527bb556f)